esphome/components/display/__init__.py

```diff
@@ -67,6 +69,18 @@ def validate_auto_clear(value):
     }
 ).extend(cv.polling_component_schema("1s"))
 
+
+def _validate_test_card(config):
+    if (
+        config.get(CONF_SHOW_TEST_CARD, False)
+        and config.get(CONF_UPDATE_INTERVAL, False) == SCHEDULER_DONT_RUN
+    ):
+        raise cv.Invalid(
+            f"`{CONF_SHOW_TEST_CARD}: True` cannot be used with `{CONF_UPDATE_INTERVAL}: never` because this combination will not show a test_card."
```

Comment:
```suggestion
            f"`{CONF_SHOW_TEST_CARD}: True` together with `{CONF_UPDATE_INTERVAL}: never` cannot be used because this will not show the test_card."
```
I would like to see this message a bit friendlier, so this is my suggestion.

Reply:
I like focusing on the fact that they entered `show_test_card: true`. I asked my partner who not technical and is a journalist and she thought the first made more sense so I'm going to stick with that.